ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -219,23 +219,11 @@ message DisclosedContract {
   // Required
   string contract_id = 2;
 
-  // The contract arguments
-  // Required
-  oneof arguments {
-    // The contract arguments as typed Record
-    // Deprecated in favor of `created_event_blob`
-    Record create_arguments = 3 [deprecated = true];
+  reserved 3; // removed create_arguments
 
-    // The contract arguments specified using an opaque blob extracted from the ``create_arguments_blob`` field
-    // of a ``com.daml.ledger.api.v1.CreatedEvent``.
-    // Deprecated in favor of `created_event_blob`
-    google.protobuf.Any create_arguments_blob = 5 [deprecated = true];
-  }
+  reserved 4; // removed metadata
 
-  // The contract metadata from the create event.
-  // Deprecated in favor of `created_event_blob`
-  // Optional
-  ContractMetadata metadata = 4 [deprecated = true];
```

Comment:
You can also kill the masseg itself no?

Reply:
We'd be loosing extensibility wouldn't we?

Reply:
by killing an unused `ContractMetadata` field? I don't see how that loses extensibilityt.

Reply:
Ah sorry, I've thought you want to remove the `DisclosedContract` message, as it now only contains a single field.

Reply:
Yes, yes. Canton repo compilation told me about this too.

Reply:
Actually, I have learned that the metadata is used in canton repair tooling and is needed for the old exports